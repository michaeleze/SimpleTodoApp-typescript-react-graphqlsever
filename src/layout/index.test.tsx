import React from 'react';
import { render, screen } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Layout from './index';
import { todoService } from '../service';

describe('Layout component', () => {
  it('should renders snapshot', () => {
    const { container } = render(<Layout />)
    expect(container).toMatchSnapshot();
  });

  it('should renders layout component', () => {
    const { getByTestId } = render(<Layout />)
    expect(getByTestId('layout')).toBeInTheDocument();
  });
});
