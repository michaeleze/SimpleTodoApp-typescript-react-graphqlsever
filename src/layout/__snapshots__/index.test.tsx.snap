// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout component should renders snapshot 1`] = `
<div>
  <div
    class="wrapper"
  >
    <div
      class="layout-container"
      data-testid="layout"
    >
      <div
        class="layout-add"
      >
        <table
          class="add-container"
          data-testid="add-task"
        >
          <tr>
            <td
              class="add-input"
            >
              <textarea
                class="add-input--field"
                data-testid="add-task-input"
                placeholder="Add new task"
              />
            </td>
            <td>
              <button
                class="edit-button"
                data-testid="create-task-input"
              >
                <svg
                  fill="currentColor"
                  height="15"
                  size="15"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 448 512"
                  width="15"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M416 208H272V64c0-17.67-14.33-32-32-32h-32c-17.67 0-32 14.33-32 32v144H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h144v144c0 17.67 14.33 32 32 32h32c17.67 0 32-14.33 32-32V304h144c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z"
                  />
                </svg>
              </button>
            </td>
          </tr>
        </table>
      </div>
      <div
        class="layout-list"
        data-testid="todo-list"
      >
        <div>
          Loading List...
        </div>
      </div>
    </div>
  </div>
</div>
`;
